src/quo2/components/text_combinations/style.cljs

```diff
@@ -2,14 +2,10 @@
 
 (defn container
   [container-style]
-  (merge
-   {:flex              1
-    :margin-horizontal 20}
-   container-style))
+  (assoc container-style :margin-horizontal 20))
```

Comment:
I think we should leave this as merge because we want to be able to overwrite the defaults imo. Otherwise the `container-style` prop does not function as expected. 
Or perhaps I have misinterpreted how `assoc` works. 
In any case `container-style` should be able to overwrite defaults

Reply:
@flexsurfer, curious of why you didn't change this? there are several uses of `container-style` prop for this component and so this can effect them.
imo it's better to set margins on the component where it's being used, otherwise we tend to be fighting with margins on the page it's being used.

Reply:
i did but in another PR https://github.com/status-im/status-mobile/pull/17215